.env.example

```diff
@@ -31,47 +30,48 @@ SQS_WORKER_TRIGGER_QUEUE_URL=
 ##### SNS #####
 
 ALERTS="sns"
-AWS_SNS_ARN="arn:aws:sns:us-east-1:000000000000:madara-orchestrator-arn"
+MADARA_ORCHESTRATOR_AWS_SNS_ARN="arn:aws:sns:us-east-1:000000000000:madara-orchestrator-arn"
 
 ##### DATABASE #####
 
 DATABASE=
```

Comment:
we had to actually delete `DATABASE`, `ALERTS` etc. i think these arne't used anymore

Reply:
basically, all env variables should have `MADARA_ORCHESTRATOR` prefix except AWS ones (which is already the case). ig the ones that don't have it can be deleted here